UVA-Relay/sql/Sqlite.cs

```diff
@@ -0,0 +1,194 @@
+                        cmd.Parameters.Add(nameParameter);
+                        cmd.Parameters.Add(idParameter);
+                        nameParameter.Value = guildName;
+                        idParameter.Value = guildId;
+                        //Execute query
+                        cmd.ExecuteNonQuery();
+                    }
+                }
+            }
+            catch (SQLiteException exc)
+            {
+                //Console.WriteLine(exc.Message);
+                Console.WriteLine("Guild already in database or error has occured");
+            }
+
+        }
+
+        //Adds users in a guild to database
+        public void AddUsersToDatabase(ulong discordId, ulong guildId)
+        {
+            var s = new SQLiteConnection(@_dbFile);
+            try
+            {
+                using (s)
+                {
+                    using (var cmd = s.CreateCommand())
+                    {
+                        s.Open();
+                        //"INSERT INTO USER (discordId, canvasId) VALUES(@discordIdQuery, @canvasIdQuery)";
+                        cmd.CommandText = "INSERT INTO USER (discordId, guildId) VALUES(@discordIdQuery, @guildIdQuery)";
+                        //Safest way to insert data without sql injections?
+                        var discordIdParameter = cmd.CreateParameter();
+                        //var canvasIdParameter = cmd.CreateParameter();
+                        var guildIdParameter = cmd.CreateParameter();
+                        discordIdParameter.ParameterName = "@discordIdQuery";
+                        guildIdParameter.ParameterName = "@guildIdQuery";
+                        //canvasIdParameter.ParameterName = "@canvasIdQuery";
+                        cmd.Parameters.Add(discordIdParameter);
+                        cmd.Parameters.Add(guildIdParameter);
+                        //cmd.Parameters.Add(canvasIdParameter);
+                        discordIdParameter.Value = discordId;
+                        guildIdParameter.Value = guildId;
+                        //canvasIdParameter.Value = ;
+                        //Execute query
+                        cmd.ExecuteNonQuery();
+                    }
+                }
+            }
+            catch (SQLiteException exc)
+            {
+                //Console.WriteLine(exc.Message);
+                Console.WriteLine("User already in database or error has occured");
+
+            }
+        }
+        //Update guild settings to database
+        public void UpdateGuildSettings(long guildId, long value, string option)
+        {
+            var s = new SQLiteConnection(@_dbFile);
+            try
+            {
+                using (s)
+                {
+                    using (var cmd = s.CreateCommand())
+                    {
+                        s.Open();
+                        cmd.CommandText = $"UPDATE GUILD SET {option} = @valueQuery WHERE GuildId = @GuildIdQuery";
```

Comment:
The option parameter should be a prepared parameter like the rest. Also, please use the cmd.Parameters.AddWithValue() method for setting prepared parameters